client/src/components/BrowseChefs/Chefs.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Grid, Paper, Box, Avatar, Typography, Chip } from "@material-ui/core";
+
+const Chefs = ({ chefProfileUrl, name, location, cuisineTags, bio }) => {
+  const useStyles = makeStyles((theme) => ({
+    avatar: {
+      width: "100px",
+      height: "100px",
+    },
+
+    cuisineChip: {
+      borderRadius: "0px",
+    },
+
+    chefBio: {
+      display: "box",
+      lineClamp: 2,
+      boxOrient: "vertical",
+      overflow: "hidden",
+    },
+  }));
+
+  const classes = useStyles();
+
+  const mapCuisineTags = (cuisineTags) => {
+    if (cuisineTags.length > 0) {
+      return cuisineTags.map(
+        (cuisineTag) => `${cuisineTag.toUpperCase()} CUISINE`
+      );
+    }
+
+    return " CUISINE";
```

Comment:
This seems like an odd default case, is it possible for a chef to not have any cuisines listed? If it is maybe just "CUISINE". Also this should be an array with one element, possibly a Chip as I say above

Reply:
the getChef api endPoint should filter all chefs who has profile with cuisines listed.